frontend/src/scenes/session-recordings/player/PlayerMetaLinks.tsx

```diff
@@ -61,13 +85,12 @@ export function PlayerMetaLinks(): JSX.Element {
     }
 
     const mode = logicProps.mode ?? SessionRecordingPlayerMode.Standard
-    const isInNotebook = !!nodeLogic
 
     return (
         <div className="flex flex-row gap-1 items-center justify-end">
             {![SessionRecordingPlayerMode.Notebook, SessionRecordingPlayerMode.Sharing].includes(mode) ? (
                 <>
-                    {isInNotebook && (
+                    {featureFlags[FEATURE_FLAGS.NOTEBOOKS] && (
```

Comment:
We have a nifty helper for this called `<FlaggedFeature` which I would recommend using